app/lib/subversion_repository.rb

```diff
@@ -32,6 +28,10 @@ def initialize(connect_string)
     end
   end
 
+  def self.permission_file
+    Settings.repository.storage + '/svn_authz'
```

Comment:
Please move this to `Settings.key_storage + '/.access' to be consistent with the git version. (Even though we aren't using keys for svn.)